cuda_phase-retrieval_class/test/test_cupy.hpp

```diff
@@ -147,4 +166,28 @@ void test_cupy_from_c_memory()
     assert(py::module::import("cupy").attr("get_default_memory_pool")().attr("used_bytes")().cast<size_t>() > 0 );
 
     //cp removed automatically after the next closing bracket
+}
+
+template<typename TInputData, typename TOutputData>
+TOutputData * convertToCUFFT(TInputData * ptr)
```

Comment:
Remove the definition. If we have no specialized version of a function, the compiler can throw a compiler error.

Reply:
Please check this. I decided to delete these two lines